docs/source/dev-guide/plugins/index.rst

```diff
@@ -56,67 +54,82 @@ a package out of the custom plugin hook shown above:
    build-backend = "setuptools.build_meta"
 
    [project]
-   name = "my-conda-plugin"
+   name = "conda-example-plugin"
    version = "1.0.0"
-   description = "My conda plugin"
-   requires-python = ">=3.7"
+   description = "Example conda plugin"
+   requires-python = ">=3.8"
    dependencies = ["conda"]
 
    [project.entry-points."conda"]
-   my-conda-plugin = "my_plugin"
-
-
-The ``setup.py`` file below is an alternative to the ``pyproject.toml`` file shown
-above; its main difference is the ``entry_points`` argument that is provided to the
-``setup()`` function:
+   conda-example-plugin = "example_plugin"
 
 .. code-block:: python
    :caption: setup.py
 
    from setuptools import setup
 
    setup(
-       name="my-conda-plugin",
+       name="conda-example-plugin",
        install_requires="conda",
-       entry_points={"conda": ["my-conda-plugin = my_plugin"]},
-       py_modules=["my_plugin"],
+       entry_points={"conda": ["conda-example-plugin = example_plugin"]},
+       py_modules=["example_plugin"],
    )
 
+In both examples shown above, we define an entry point for conda. It's important to make sure
+that the entry point is for "conda" and that it points to the correct module in your plugin package.
```

Comment:
Would we want to put "conda" in backticks here?

Reply:
Since this section is talking about entry points and we're referring to conda as the specific example, I think the quote marks are appropriate.

Reply:
Sorry, just saw this. I meant doing `"conda"`, to be clear. But no worries either way.